general-superstaq/general_superstaq/service.py

```diff
@@ -1,14 +1,17 @@
 import os
-from typing import Any, Dict, Optional, Tuple, Union
+from typing import TYPE_CHECKING, Any, Dict, Optional, Tuple, Union
 
-import general_superstaq as gss
+import qubovert as qv
 
+if TYPE_CHECKING:
```

Comment:
```suggestion
import general_superstaq as gss
```

Reply:
I thought the plan was to deploy this and then make the changes to the MIS notebook since the kernel in the jupyter notebook requires the change to be made in the main branch. What do you think?

Reply:
I'm only talking about testing here. You can run the MIS notebook from this branch but don't commit the notebook.

Reply:
Sounds good, we will address this in the next PR.